File: src/IsbInterop.CSharp/Data/IPickRequisite.cs
﻿namespace IsbInterop.Data
{
  /// <summary>
  /// Реквизит типа «Признак».
  /// </summary>
  public interface IPickRequisite : IRequisite
  {
    /// <summary>
    /// Cписок описаний допустимых значений реквизита.
    /// </summary>
    IPickRequisiteItems Items { get; }
  }
}

File: src/IsbInterop/Data/IPickRequisite.vb
Namespace Data

  ''' <summary>
  ''' Реквизит типа «Признак».
  ''' </summary>
  Public Interface IPickRequisite
    Inherits IRequisite

    ''' <summary>
    ''' Cписок описаний допустимых значений реквизита.
    ''' </summary>
    ReadOnly Property Items As IPickRequisiteItems
  End Interface
End NameSpace